### ISSUE TITLE
Navigating back from Keyguard does not always work

### ISSUE THREAD

**Issue by sisou** [nimiq/accounts#75]
The Keyguard client is affected by https://github.com/nimiq/rpc/issues/5.

This happens when using a request that is directly handed through the accounts manager to the keyguard.

**Comment by sisou** [comment]
This might be a valid fix, but I could not test it, because with #106 this has not happend anymore.

E.g. for [Login.vue](https://github.com/nimiq/accounts/blob/master/src/views/Login.vue) we could replace the current state:
```ts
@Component
export default class Login extends Vue {
    @Static private request!: ParsedLoginRequest;
    public created() {
        const request: ImportRequest = {
            appName: this.request.appName,
            defaultKeyPath: DEFAULT_KEY_PATH,
            requestedKeyPaths: [DEFAULT_KEY_PATH],
        };

        const client = this.$rpc.createKeyguardClient();
        client.import(request).catch(console.error); // TODO: proper error handling
    }
}
```
With a `beforeRouteEnter` hook, whose `next` callback [gets called after the routing finished](https://router.vuejs.org/guide/advanced/navigation-guards.html#the-full-navigation-resolution-flow):
```
@Component({
    beforeRouteEnter: function(to: any, from: any, next: any) {
        console.log("beforeRouteEnter called");
        next((vm: any) => {
            console.log("beforeRouteEnter callback called");

            const request: ImportRequest = {
                appName: vm.request.appName,
                defaultKeyPath: DEFAULT_KEY_PATH,
                requestedKeyPaths: [DEFAULT_KEY_PATH],
            };

            const client = vm.$rpc.createKeyguardClient();
            vm.$router.afterEach((to: any, from: any) => console.log("Finished navigation from", from, "to", to));
            client.import(request).catch(console.error); // TODO: proper error handling
        });
    }
})
export default class Login extends Vue {
    @Static private request!: ParsedLoginRequest;
}
```

**Comment by sisou** [comment]
Closed as it has not happend anymore recently.
